include/seqan3/argument_parser/validators.hpp

```diff
@@ -613,24 +621,41 @@ class output_file_validator : public file_validator_base
      */
 
     //!\copydoc seqan3::input_file_validator::input_file_validator()
-    output_file_validator()
+    output_file_validator() = default; //!< Defaulted.
```

Comment:
```suggestion
    output_file_validator() : output_file_validator{output_file_open_options::create_new}
    {}
```

? This would enable the extension list again, if we provide `file_t` in the default case.

Reply:
Yes, I re-changed the case above again.